### PR TITLE
Redesign electrode-webpack-reporter and incorporate electrify

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

**Comment by ananavati** [comment]
@georgeweiler 

We've got a file name case issue here on Travis CI. It works on OSX but not *nix because of the file system file name sensitivity.

https://travis-ci.org/electrode-io/electrode-webpack-reporter/builds/176544470

Module not found: Error: Cannot resolve 'file' or 'directory' ./electrify-utils/d3Visualization in /home/travis/build/electrode-io/electrode-webpack-reporter/client/components/webpack-modules
 @ ./client/components/webpack-modules/electrify.jsx 12:1032-1076

Please make sure the filenames are case-sensitive.

File: client/components/home.jsx [hunk 1]
```diff
@@ -1,70 +1,45 @@
 import React, {PropTypes} from "react";
 import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
-import {Card, CardHeader, CardText} from "material-ui/Card";
+import darkBaseTheme from "material-ui/styles/baseThemes/darkBaseTheme";
```
Comment (ananavati) [inline client/components/home.jsx:3]: naming convention `darkBaseTheme` => `DarkBaseTheme`

File: client/components/home.jsx [hunk 2]
```diff
@@ -1,70 +1,45 @@
...
+import getMuiTheme from "material-ui/styles/getMuiTheme";
```
Comment (ananavati) [inline client/components/home.jsx:4]: naming convention `getMuiTheme ` => `GetMuiTheme `

File: client/styles/base.css [hunk 3]
```diff
@@ -1,18 +1,61 @@
+* {
+  -webkit-box-sizing: border-box;
```
Comment (ananavati) [inline client/styles/base.css:2]: code styling messed up with spaces in this block

File: client/styles/base.css [hunk 4]
```diff
@@ -1,18 +1,61 @@
...
+  letter-spacing: 0.075em;
+  border-bottom: 1px rgba(255,255,255,0.5);
+  padding-top: 0.775em;
+  box-shadow: 0 11px 10px -10px rgba(0,0,0,0.25);
 }
 
-table {
-    border-collapse: collapse;
-    border: dashed 1px;
+.container {
```
Comment (ananavati) [inline client/styles/base.css:31]: same here, please make sure the whole file has consistent tabs/spaces

**Comment by ananavati** [comment]
a few more comments, otherwise 👍

File: client/components/home.jsx (hunk 2 above)
Comment (georgeweiler) [reply client/components/home.jsx:4]: @ananavati, is it convention to capitalize *all* imports? darkBaseTheme and getMuiTheme are not react components.

**Comment by jchip** [comment]
👍  Just a tech-debt - passing the entire webpack stats as pureWebpackStats is wasteful.  Potentially duplicates with info in modulesByPkg.  Loading all these is probably very slower now, even on localhost.

**Comment by georgeweiler** [comment]
You're right Joel. I will look at a way to avoid duplicating the webpack info.

**Comment by ananavati** [comment]
merging this pr and adding this ticket for enhancing the webpack stats object 
issue: https://github.com/electrode-io/electrode-webpack-reporter/issues/11

**Pull request merged**